src/marten/cli/manage/command/new/templates/app/shard.override.yml.ecr

```diff
@@ -0,0 +1,4 @@
+dependencies:
+  marten:
+    github: martenframework/marten
+    branch: main
```

Comment:
Since the content of this file is going to be the same for both app and project repository structures, we could define it only once in the `src/marten/cli/manage/command/new/templates/shared/` folder.